modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -191,6 +196,9 @@ public class RocksDbKeyValueStorage implements KeyValueStorage {
     /** Revision to timestamp mapping column family. */
     private volatile ColumnFamily revisionToTs;
 
+    /** Revision to checksum mapping column family. */
+    private volatile ColumnFamily revisionToChecksum;
```

Comment:
Let's check how it will work with snapshots in tests:
1. org.apache.ignite.internal.metastorage.server.BasicOperationsKeyValueStorageTest#testSnapshot
2. org.apache.ignite.internal.metastorage.server.BasicOperationsKeyValueStorageTest#testClearDataBeforeRestoreFromSnapshot

Reply:
I added the corresponding check to the first test you mention, but not to the second one (as it's not clear how it could be implemented without looking at the storage internals)

Reply:
Ok